Library/Homebrew/cask/cask_loader.rb

```diff
@@ -301,11 +301,7 @@ def load(config:)
 
           json_cask[:artifacts].each do |artifact|
             key = artifact.keys.first
-            if FLIGHT_STANZAS.include?(key)
-              instance_eval(artifact[key])
-            else
-              send(key, *artifact[key])
-            end
+            send(key, *artifact[key])
```

Comment:
```suggestion
            send(key, *artifact[key]) if artifact[key].present?
```

Is this needed?

Reply:
Why would that be needed? You can't reach this code if you use a flight stanza.

Reply:
You shouldn't make it past the following code if you have a flight artifact.

https://github.com/Homebrew/brew/blob/2a417f14163aacfb4ffcb754fe9dea71c46f6ad0/Library/Homebrew/cask/cask_loader.rb#L233-L238